lua-resty-s3-0.1-1.rockspec

```diff
@@ -0,0 +1,24 @@
+package = "lua-resty-s3"
+version = "0.1-1"
+source = {
+   url = "git+https://github.com/OnlyPiglet/lua-resty-s3.git"
```

Comment:
Shouldn't this PR link be linked to my repository?

Reply:
sorry I forget to change this , i will fix this and upload again

Reply:
> Shouldn't this PR link be linked to my repository?

i am sorry for forgeting changing this url ,I will fix this and uplaod again

Reply:
> Shouldn't this PR link be linked to my repository?

hello I have changed the source.url of rockspec and I have upload this package on rock hub, the url is https://luarocks.org/modules/OnlyPiglet/lua-resty-s3